addons-l10n/en/search-profile.json

```
@@ -1,4 +1,4 @@
 {
   "search-profile/profile": "Profile",
-  "search-profile/invalid-username": "\"{username}\" is not a valid username"
+  "search-profile/invalid-username": "\"{username}\" is not a valid username or the account doesn't exist"
```

```suggestion
  "search-profile/invalid-username": "\"{username}\" is not a valid username or the account does not exist"
```
Using "is not" and "doesn't" in the same sentence sounds weird.